public/app/core/services/keybindingSrv.ts

```diff
@@ -39,9 +39,11 @@ export class KeybindingSrv {
       this.bind('g a', this.openAlerting);
       this.bind('g p', this.goToProfile);
       this.bind('g e', this.goToExplore);
-      this.bind('s o', this.openSearch);
+      if (!config.featureToggles.topNavCommandPalette) {
+        this.bind('s o', this.openSearch);
+        this.bind('f', this.openSearch);
+      }
       this.bind('t a', this.makeAbsoluteTime);
-      this.bind('f', this.openSearch);
```

Comment:
I would rather we keep this (or change it to `s`) , seems like a much easier / better shortcut for command palette than ctrl+k  
GitHub has `s` or `/`  for search

Reply:
think there's 2 options:
1. stick with `cmd+k`
1. pick something else (e.g. `s`), display that in the top search input **but** also keep `cmd+k` as an undocumented shortcut as well since it's well known in other contexts

in our mob we settled on `cmd+k` for a couple of reasons:
- it's a known pattern already
- we've had github/support issues in the past with single key shortcuts where users have triggered them accidentally by typing when not properly focused on an input. this can be quite jarring and is very unlikely to happen with `cmd+k`

having said that, i'm not completely opposed to option 2 above if you feel strongly about it. 😅 

cc @amy-super @grafana/user-essentials

Reply:
cmd+k just feels very uncommon for search, but I mind so much :) 

I think we need to expand the width of the "input" a bit 
<img width="826" alt="Screenshot 2023-01-27 at 16 12 11" src="https://user-images.githubusercontent.com/10999/215120193-66a5cb40-512b-417d-aae9-20469d97e4b2.png">

Reply:
yeah that's true, it's not a common search shortcut but is a common command palette shortcut. i'll fix the input width now 👍